### PR TITLE
fix: MD049 consistent emphasis style

### PR DESCRIPTION
**Pull request by nschonni**
Second batch of asterisks to underscore for emphasis style. There is some edge cases with this style sometimes when you're trying to do emphasis inside a word

### CODE REVIEW

File: files/en-us/web/svg/attribute/begin/index.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ The definition of `<begin-value-list>` is the [same as for the other animation e
 
 {{EmbedLiveSample('Accesskey_example', '100%', 130)}}
 
-*This example is embed in an iFrame. If you want to activate the key events, you have to click on it first.*
+_This example is embed in an iFrame. If you want to activate the key events, you have to click on it first._
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/begin/index.md:322]:
```suggestion
_This example is embedded in an iFrame. If you want to activate the key events, you have to click on it first._
```

File: files/en-us/web/svg/attribute/end/index.md [hunk 2]
```diff
@@ -189,7 +189,7 @@ The `<end-value-list>` is a semicolon-separated list of values. Each value can b
 
 {{EmbedLiveSample('Accesskey_example', '100%', 130)}}
 
-*This example is embed in an iFrame. If you want to activate the key events, you have to click on it first.*
+_This example is embed in an iFrame. If you want to activate the key events, you have to click on it first._
```
Comment (wbamberg) [inline files/en-us/web/svg/attribute/end/index.md:192]:
```suggestion
_This example is embedded in an iFrame. If you want to activate the key events, you have to click on it first._
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Looks great. I suggested a couple of grammar fixes (for errors that you did not introduce, but that are in lines you touched).

File: files/en-us/web/svg/attribute/begin/index.md (hunk 1 above)
Comment (nschonni) [inline files/en-us/web/svg/attribute/begin/index.md:322]:
```suggestion
_This example is embedded in an iFrame. If you want to activate the key events, you have to click on it first._
```

**Review by wbamberg — APPROVED** [review]
👍 nice!

**Pull request merged**